mini.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styleHome.css" />
+    <title>Lena's ZOL</title>
```

Comment:
Il manque la balise meta description indispensable pour le référencement comme par exemple : 
<meta name="description" content="This is an example of a
meta description. This will often show up in search results.">